MainModule/Client/Client.lua

```diff
@@ -800,7 +800,7 @@ return service.NewProxy({
 		log("~! Return success")
 		return "SUCCESS"
 	end,
-	__metatable = "Adonis",
+	__metatable = Folder:FindFirstChild("ADONIS_DEBUGMODE_ENABLED") and nil or unique,
```

Comment:
I'll stop marking these, but all of these statements need to be converted to if-else